cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -143,46 +144,46 @@ def count_records_rollups_table(db, provider):
 
 
 @pytest.fixture(scope="module")
-def resource_usage(vm_ownership, db, provider, ssh_client_modscope):
+def resource_usage(vm_ownership, appliance, provider):
     # Retrieve resource usage values from metric_rollups table.
     average_cpu_used_in_mhz = 0
     average_memory_used_in_mb = 0
     average_network_io = 0
     average_disk_io = 0
     vm_name = provider.data['cap_and_util']['chargeback_vm']
 
-    metrics = db['metrics']
-    rollups = db['metric_rollups']
-    ems = db['ext_management_systems']
+    metrics = appliance.db['metrics']
+    rollups = appliance.db['metric_rollups']
+    ems = appliance.db['ext_management_systems']
     logger.info('DELETING METRICS DATA FROM METRICS AND METRIC_ROLLUPS tables')
-    db.session.query(metrics).delete()
-    db.session.query(rollups).delete()
+    appliance.db.session.query(metrics).delete()
+    appliance.db.session.query(rollups).delete()
 
-    provider_id = db.session.query(ems).filter(ems.name == provider.name).first().id
+    provider_id = appliance.db.session.query(ems).filter(ems.name == provider.name).first().id
 
-    # Chargeback reporting is not done on real-time values.So, we are capturing C&U data
-    # and forcing hourly rollups by running these commands through the Rails console.
+    # Chargeback reporting is done on rollups and not  real-time values.So, we are capturing C&U
+    # data and forcing hourly rollups by running these commands through the Rails console.
 
     logger.info('CAPTURING PERF DATA FOR VM {} running on {}'.format(vm_name, provider.name))
-    ssh_client_modscope.run_rails_command(
+    store.current_appliance.ssh_client.run_rails_command(
```

Comment:
REQUIRED: you got appliance from appliance fixture. Please replace current_appliance with appliance.